### PR TITLE
initiate the titan spam

### PR DESCRIPTION
**Pull request by ThatOneBepis**

### CODE REVIEW

File: content/blocks/titan-factory.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+"maxUnits": 5
```
Comment (Anuken) [inline content/blocks/titan-factory.json]: Needs a 4-space indent here.

**Comment by ZkyLB** [comment]
nice

**Comment by Serdjio3** [comment]
Is it possible to make a factory, which have 2 or above items in output?

**Comment by Anuken** [comment]
@Serdjio3 No.

**Pull request closed without merging**